client/src/pages/Events/Events.spec.tsx

```diff
@@ -0,0 +1,65 @@
+import { render } from "@testing-library/react";
+import { Events } from "./Events";
+import { EventsPageObject } from "./Events.pageObject";
+import * as getEventsUtils from "./useFetchEvents";
+
+describe("Events component", () => {
+  const renderComponent = () => {
+    return new EventsPageObject(render(<Events />));
```

Comment:
tady je treba to "zabalit" do komponenty `BrowserRouter`, jinak react-router haze chybu
```
import { BrowserRouter } from "react-router";

describe("Events component", () => {
  const renderComponent = () => {
    return new EventsPageObject(render(<BrowserRouter><Events /></BrowserRouter>));
  };
```